### ISSUE TITLE
Increasing the number of results when using Vomnibar

### ISSUE THREAD

**Issue by v1kn** [philc/vimium#2708]
`Vimium 1.60.4 on Firefox 56`

When using `t`/`T` or `b`/`B`, the results seem to be always limited to 10. Instead of scrolling down, the selector moves back to the top. Is there a way to increase the results count and add scrolling, so that I can search for a given phrase in all my bookmarks?

**Comment by smblott-github** [comment]
> Is there a way to increase the results count and add scrolling

Not currently.

**Comment by v1kn** [comment]
OK, but limiting the results to 10  can significantly decrease usability with greater bookmark count.
Please consider lifting this limit, as long as there are no API obstacles

**Comment by philc** [comment]
When I first read this proposal, I thought "it seems reasonable to want to show more than 10 results.". Sometimes there's a lot of junk in my history which clutters up the results, and I've wondered if the real result I want is #11 or beyond.

However, these are the counter arguments:

1. I've never heard of people complaining about this.
2. The reason I've never had an issue with only 10 results myself is because the design behind the vomnibar is to "keep typing until you see the result you want". So if you have more than 10 results, typing a few more characters should narrow the result set event further.
3. In fact, we directly mimicked the design of Chrome's omnibar when implementing Vimium's vomnibar. Chrome shows just 10 results, so there's some strong design precedent that this is sufficient for most use cases.

I'm inclined to keep it the way it is unless there's some important use case that we, and the Chrome team, missed with the design of this UX, which can't be satisfied simply by "typing more characters".

@v1kn, @onriv, any additional commentary?

**Comment by onriv** [comment]
As for me, I agree with that the vomnibar is used for "keep typing until you see the result you want". But there were cases that I tried my best to keep typing, and there were still much more than 10 results shown up (for example, I typed something like "Java" or "Python"). So I moved up and down in the results to get more hints to get some keywords that could maybe help me narrowing the results or the result I wanted directly.

Yeah I agree with that increasing the number of results would change the aspiration for the vomnibar. Since tried it in my chrome, I did care less about the keywords typed. And I dont konw why recently the omnibar has some delay to show up in a webpage (delay at the first time at each webpage, after that it's ok). And I have less expierence with using the vomnibar now (now in most cases I just try typing something at the address bar in chrome). The increasing number may do some effect on performance.

So I agree with keeping it with 10 too.
